lib/games_manager.rb

```diff
@@ -46,8 +46,9 @@ def percentage_visitor_wins
     ratio(wins, total_games)
   end
 
-  def total_games
-    @games.count
+  def total_games(filtered_games = @games)
+    filtered_games.count
   end
```

Comment:
Let's keep total_games stripped down for now. Use the method season_group to count games by season.

Reply:
Joshua, I need these arguments to calculate my averages on a varying set of game records.  If you have an alternate solution, i'm open to ideas.